editor/src/clj/editor/code/view.clj

```diff
@@ -3735,13 +3758,6 @@
         canvas-pane (Pane. (into-array Node [canvas]))
         undo-grouping-info (pair :navigation (gensym))
         lsp (lsp/get-node-lsp basis resource-node)
-        completions-shortcut-text (transduce
-                                    (comp
-                                      (mapcat val)
-                                      (filter #(= :proposals (:command %)))
-                                      (map #(.getDisplayText ^KeyCombination (:key-combo %))))
-                                    util/first-rf
-                                    (g/node-value app-view :keymap))
```

Comment:
Now we pipe the stuff through the graph so that setting a `:keymap` property on the `app-view` node refreshes the completion shortcut text.